include/taskolib/Context.h

```diff
@@ -79,7 +79,7 @@ struct Context
     VariableTable variables;
 
     /// An initialization function that is called on a LUA state before a step is executed.
-    std::function<void(sol::state&)> lua_init_function;
+    std::function<void(sol::state&)> lua_step_setup;
```

Comment:
Would not be `lua_setup` more fitting?

This name clashes in my head with `step_setup` later on (which is in another object admittedly. But I also not that it _should/could_ be in the Context instead and then it would directly clash.

But even in different objects, the names and purposes are too similar for me.

Reply:
Maybe step_setup_function (C++) as opposed to step_setup_script (Lua)? It remains a bit confusing...

Reply:
I change it to `step_setup_function`. Maybe change in other PR.

Reply:
... can it be that you did not push your changes to Github yet? I still see `lua_step_setup`.